### ISSUE TITLE
Another MD Issue Discovered

### ISSUE THREAD

**Issue by tundratim** [gitbucket/gitbucket#1664]
The following markdown does not render a horizontal rule line as it should:

    ````
    This is a paragraph
    --------------------------
    This is a paragraph after a horizontal rule

It appears that the `<hr>` tag is not even being generated in this case.

Note that this does work here on github.

**Comment by takezoe** [comment]
Thanks for your reporting. `<hr>` has been filtered by a security filter in [markedj](https://github.com/gitbucket/markedj), I added it to the white list in https://github.com/gitbucket/markedj/commit/19620e8efb681d9d23b3a24464e9f8caf7dff7d2.

However, in that case, GitHub seems to render as headline like this:

This is a paragraph
--------------------------
This is a paragraph after a horizontal rule

**Comment by tundratim** [comment]
Confirmed fixed in 4.16.0
